enterprise_access/apps/content_assignments/utils.py

```diff
@@ -44,13 +53,15 @@ def get_self_paced_normalized_start_date(start_date, end_date, course_metadata):
     default to today's date.
     Otherwise, return the current start_date
     """
-    curr_date = datetime.now()
+    curr_date = localized_utcnow()
     pacing_type = course_metadata.get('pacing_type', {}) or None
     weeks_to_complete = course_metadata.get('weeks_to_complete', {}) or None
     if not (start_date and end_date and pacing_type and weeks_to_complete):
         return curr_date.strftime(BRAZE_TIMESTAMP_FORMAT)
-    if pacing_type == "self_paced":
-        if has_time_to_complete(curr_date, end_date, weeks_to_complete) or \
-                is_within_minimum_start_date_threshold(curr_date, start_date):
+    start_date_datetime = parser.parse(start_date).astimezone(UTC)
+    end_date_datetime = parser.parse(end_date).astimezone(UTC)
+    if pacing_type == "self_paced" and start_date_datetime < curr_date:
```

Comment:
This one line is the meat of the fix, the rest are code cleanup / readability improvement.